data/mods/Russ_Extra_Stuff/items/r_guns.json

```diff
@@ -0,0 +1,242 @@
+[
+
+{
+    "id": "alecto_airpistol",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "Webley Alecto .22 air pistol",
+    "name_plural": "Webley Alecto air pistols",
+    "description": "The Alecto .22 is a match grade multi-pump pneumatic air pistol.  It features 3 power levels - topping out at over 400 feet per second, 4-way adjustable trigger, ergonomic grip, accessory rails for mounting scopes, recoil free shooting, a rifled steel barrel, adjustable front & rear sights.  It takes a little longer to load because of the need for 3 pumps.",
+    "material": ["steel", "null"],
+    "skill": "pistol",
+    "price": 25000,
+    "ammo": "22air",
+    "weight": 976,
+    "volume": 6,
+    "bashing": 8,
+    "cutting": 0,
+    "to_hit": 1,
+    "ranged_damage": 5,
+    "dispersion": 20,
+    "sight_dispersion": 40,
+    "flags": ["RELOAD_ONE" , "NEVER_JAMS", "STR_RELOAD"],
+    "aim_speed" : 4,
+    "recoil": 0,
+    "durability": 5,
+    "clip_size": 1,
+	"range":18,
+    "reload": 90,
+    "loudness" : -20,
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "mechanism", 2 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ], ["spring",1], ["screws",1] ]
+  },
+    {
+    "id": "buckmark_airpistol",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "Browning Buckmark air pistol",
+    "name_plural": "Crosman 1322s",
+    "description": "A popular spring-piston break barrel .177 air pistol, the Browning Buckmark was popular due to it's easy handling, high accuracy, and shooting enjoyment.  Shoots at around 350 fps in utter silence.",
+    "price": 45000,
+    "material": ["steel", "plastic"],
+    "skill": "pistol",
+    "ammo": "177air",
+    "weight": 700,
+    "volume": 3,
+    "bashing": 9,
+    "cutting": 0,
+    "to_hit": 0,
+    "ranged_damage": 4,
+    "dispersion": 55,
+    "sight_dispersion": 45,
+    "flags": ["RELOAD_ONE" , "NEVER_JAMS"],
+    "aim_speed" : 4,
+    "recoil": 0,
+    "durability": 6,
+    "clip_size": 1,
+	"range":14,
+    "reload": 70,
+    "loudness" : -20,
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ], ["spring",1], ["screws",1] ]
+  },{
+    "id": "crosman_1322_airpistol",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "Crosman 1322 air pistol",
+    "name_plural": "Crosman 1322s",
+    "description": "A popular multipump pneumatic .22 air pistol, the crosman 1322 is cheap but effective and popular.  It's known for being highly moddable with high quality factory iron-sights.  This one comes with a collapsable stock.  Shoots at around 600 fps in utter silence.  the reload is slower because it must be pumped ten times with the underbarrel pump arm to be effective.",
```

Comment:
A popular <...> pistol <...> is <...> popular.
Are you sure it's so popular you need to say about that twice in one sentence? (just kidding)